consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGatewayTest.java

```diff
@@ -62,12 +77,32 @@ void shouldInvokeBreathingSpace() throws Exception {
         verify(individualRequest).setRequestStatus(ProcessingStatus.PROCESSED.name());
     }
 
-    private void setupMockBehaviour() {
-        BreathingSpaceRequest breathingSpaceRequest = mock(BreathingSpaceRequest.class);
-        when(individualRequest.getRequestPayload()).thenReturn(XML);
-        try {
+    @Test
+    void shouldInvokeClaimStatusUpdate() throws Exception {
+        setupMockBehaviour(CLAIM_STATUS_UPDATE);
+        ClaimStatusUpdateResponse response = new ClaimStatusUpdateResponse();
+        response.setProcessingStatus(ProcessingStatus.PROCESSED);
+        when(claimStatusUpdate.claimStatusUpdate(anyString(), anyString(), any())).thenReturn(response);
+
+        cmcConsumerGateway.individualRequest(individualRequest, CONNECTION_TIME_OUT, RECEIVE_TIME_OUT);
+
+        verify(claimStatusUpdate).claimStatusUpdate(anyString(), anyString(), any(ClaimStatusUpdateRequest.class));
+        verify(xmlToObject).convertXmlToObject(anyString(), any());
+        verify(individualRequest).getRequestPayload();
```

Comment:
Add lines to verify individualRequest.getRequestType(), individualRequest.getSdtRequestReference() and individualRequest.setRequestStatus(ProcessingStatus.PROCESSED.name()).

Reply:
done